modules/utilities.py

```diff
@@ -661,7 +664,8 @@ def problem_widget(name,
                    problem_id,
                    disable_todo=False,
                    anchor=True,
-                   request_vars={}):
+                   request_vars={},
+                   page_prefix=None):
```

Comment:
Is the default argument required?

Reply:
Maybe default to `unknown` and add the same for GA to monitor if it creeps in in the future